src/Windowing/Silk.NET.Windowing.Glfw/GlfwWindow.cs

```diff
@@ -326,12 +326,13 @@ protected override void CoreInitialize(WindowOptions opts)
                 _glfw.WindowHint(WindowHintBool.OpenGLDebugContext, true);
             }
 
-            // Set API profile
-            _glfw.WindowHint
-            (
-                WindowHintOpenGlProfile.OpenGlProfile,
-                opts.API.Profile == ContextProfile.Core ? OpenGlProfile.Core : OpenGlProfile.Compat
-            );
+            if((opts.API.Version.MajorVersion == 3 && opts.API.Version.MinorVersion >= 2) || opts.API.Version.MajorVersion > 3)
+                // Set API profile
+                _glfw.WindowHint
+                (
+                    WindowHintOpenGlProfile.OpenGlProfile,
+                    opts.API.Profile == ContextProfile.Core ? OpenGlProfile.Core : OpenGlProfile.Compat
+                );
```

Comment:
Usually lenient on style but this is a small PR, happy to merge after this quick fix is applied ;)

```suggestion
            if ((opts.API.Version.MajorVersion == 3 && opts.API.Version.MinorVersion >= 2) || opts.API.Version.MajorVersion > 3)
            {
                // Set API profile
                _glfw.WindowHint
                (
                    WindowHintOpenGlProfile.OpenGlProfile,
                    opts.API.Profile == ContextProfile.Core ? OpenGlProfile.Core : OpenGlProfile.Compat
                );
            }
```